rustls/tests/common/mod.rs

```diff
@@ -385,7 +385,7 @@ impl KeyType {
         )))
     }
 
-    pub fn get_certified_key(&self) -> Result<Arc<CertifiedKey>, Error> {
+    pub fn get_certified_key_with_raw_pub_key(&self) -> Result<Arc<CertifiedKey>, Error> {
```

Comment:
Nit: pre-existing issue, but if we're renaming anyway, please get rid of `get_` prefixes for these renamed methods (per [API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)).